src/component.coffee

```diff
@@ -390,15 +392,13 @@ Ember.AddeparMixins.ResizeHandlerMixin,
 
   # rows that were selected directly or as part of a previous
   # range selection (shift-click)
-  persistedSelection: Ember.computed ->
-    Ember.ArrayProxy.createWithMixins Ember.MutableArray, {content:[]}
+  persistedSelection: Ember.computed -> []
```

Comment:
You should use `Ember.A()` here if we care about this working for people with Ember's prototype extensions disabled